package/yast2-schema-micro.spec

```diff
@@ -17,18 +17,21 @@
 
 
 Name:           yast2-schema-micro
-Version:        4.4.8
+Version:        4.4.9
 Release:        0
 
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
-Source0:        yast2-schema-%{version}.tar.bz2
+# source to generate rpm is same as with default, just limited set of build depedencies
+# results in limited profile
+Source0:        yast2-schema-default-%{version}.tar.bz2
 
 URL:            https://github.com/yast/yast-schema
 
 # provide schema, so it is full replacement
 Provides:       yast2-schema
 # both cannot be used as it uses same files
-Conflicts:      yast2-schema
```

Comment:
The correct would be:

Conflicts:      otherproviders(yast2-schema)

in all yast2-schema-* packages, that will be future-proof. See the -release packages for base products which use that.

Reply:
great, I do not know that otherproviders